en/docs/learn/analytics/managing-dashboard-permissions.md

```diff
@@ -1,6 +1,6 @@
 # Managing Analytics Dashboard Permissions
 
-You can view the API statistics related to both the API Publisher and the Developer Portal using the **dashboard** runtime of the WSO2 API-M Analytics server.
+You can view the API statistics related to both the API Publisher and the Developer Portal using the **dashboard** runtime of the WSO2 API Manager Analytics server.
```

Comment:
what is the "**dashboard** runtime"? Don't we call t the analytics dashboard? If not, how do we set it up? Shall we link to the place where we can set it up? We need to add it as a pre requisite to the steps as well.

Reply:
API Manager Analytics has two runtime profiles, namely dashboard and worker. The worker receives the events and does the necessary summarizations and dashboard profile used to visualize the statistics. Yes, both dashboard runtime and analytics dashboards are interchangeable. Better to use the term analytics dashboard. For this case don't think we have to add that as a pre-requisite.

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/1697/commits/ab12d89bb02874fd1eccd6b48d0a387a8ccd7ffa